src/client/app/model/arrow.ts

```diff
@@ -1,23 +1,24 @@
 import { Point } from './geometry';
+import { Card } from './card';
 import { CollaborativeObjectModel } from './collaborative-object-model';
 
 export class Arrow extends CollaborativeObjectModel {
   protected static modelName = 'Arrow';
   // Collaborative fields
   tailPosition: Point;
   tipPosition: Point;
-  fromCardId: string;
-  toCardId: string;
+  fromCard: Card;
+  toCard: Card;
```

Comment:
Nice, I didn't know that custom objects could nest references to each other and still reload just fine.

I also like the type-safety here.

Reply:
Yeah, I'm not sure how it serializes when it goes to Realtime, but it works. Lol